lib/types/wdio.d.ts

```diff
@@ -0,0 +1,79 @@
+import { Argument, Tag } from '@wdio/reporter';
+import { WriteStream } from 'fs';
+import { cjson_metadata } from '../models';
+
+declare module '@wdio/types' {
```

Comment:
Can we just extend the types in the WebdriverIO project rather then redeclaring them here?

Reply:
I have substituted all the modified types with extended ones